src/cgame/cg_local.h

```diff
@@ -3372,7 +3372,7 @@ void CG_UpdateFlamethrowerSounds(void);
 void CG_InitLocalEntities(void);
 localEntity_t *CG_AllocLocalEntity(void);
 localEntity_t *CG_FindLocalEntity(int index, int sideNum);
-void CG_AddLocalEntities(void);
+void CG_AddLocalEntities(qboolean master);
```

Comment:
I'd name this something more specific than `master`, or at least add a comment what it signifies. Somebody's going to look at this in few months and wonder what the hell is this parameter.

Reply:
^

Reply:
Added comment description to latest commit